MOD/Witcher/history/characters/065000_elves.txt

```diff
@@ -51,7 +51,7 @@
 	religion=dana_meadbh culture=aen_seidhe
 	add_trait=elf_race add_trait=proud add_trait=canon
 	give_nickname=nick_fidhail
-	1009.1.1={ birth=yes }
+	1029.1.1={ birth=yes }

```

Comment:
Could you add a comment in the file that birth date is chosen so that he has <256yo in 1274 ?

Reply:
Will do.  I'll add a comment for this.